test/sysconfig_test.rb

```diff
@@ -9,6 +9,23 @@
   subject(:sysconfig) do
     new_sysconfig(configfiles)
   end
+  let(:configfiles) { ["#{DATA_PATH}/sysconfig/*"] }
+
+  before do
+    allow(Yast::SCR).to receive(:Execute).with(path(".target.bash_output"), anything)
+      .and_return("exit" => 0, "stdout" => "", "stderr" => "")
+    # run git version of parse_configs.pl
+    allow(Yast::SCR).to receive(:Execute).with(path(".target.bash"), /parse_configs.pl/) do |path, command|
+      params = command.split
+      params[0] = SRC_PATH + "/bin/parse_configs.pl"
+      `#{params.join(" ")}`
+    end
+
+    allow(Yast::SCR).to receive(:Read).and_call_original
+    # read properly sysedit agent from git
+    allow(Yast::SCR).to receive(:Read).with(path(".target.string"), /sysedit.agent/)
+      .and_return(::File.read(SRC_PATH+"/data/sysedit.agent"))
```

Comment:
Missing spacesaround `+`, Rubocop would complain... :wink: 

But again, `File.join` would be a bit better:
```suggestion
      .and_return(::File.read(File.join(SRC_PATH, "/data/sysedit.agent")))
```